app/pages/project/field-guide-container.jsx

```diff
@@ -39,7 +40,9 @@ export default class FieldGuideContainer extends React.Component {
       return (
         <Pullout className="field-guide-pullout" side="right" open={this.state.revealed}>
           <button type="button" className="field-guide-pullout-toggle" onClick={this.toggleFieldGuide}>
-            <strong>Field guide</strong>
+            <strong>
+              <Translate content="project.classifyPage.fieldGuide" />
```

Comment:
Really minor comment about naming, but the Field Guide appears in Talk too, so isn't strictly part of the classify page.

Reply:
Ack I had forgotten the Field Guide is also in Talk, good catch! I moved it to the general project object, but open to suggestion.